### DISCUSSION TITLE
extensionsupdate log

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#329] [Q&A (Community Support)]
I have a question about the extensionsupdate.1.log files generated by Akeeba Panopticon. 
The log files are have the name of the site they belong to (I guess that is correct), 
![Akeeba-Panopticon-2](https://github.com/akeeba/panopticon/assets/17553018/0e210f3b-798a-46ad-adf0-04cf0887d260)

but when I open them, I see logs from other sites as well.
Thanks in advance for any help.

**Comment by nikosdion** [comment]
This is actually a rotated log. Hm. I think I need to change the log naming to avoid collisions.

**Reply by nikosdion** [reply]
Tracking in https://github.com/akeeba/panopticon/issues/398
